### DISCUSSION TITLE
Quay.io repository empty

### DISCUSSION THREAD

**Discussion by mOewa** [containers/podman#19797] [General]
Hello,

The repository [there](https://quay.io/repository/podman/stable?tab=tags) is empty and the [history](https://quay.io/repository/podman/stable?tab=history) show deletion of all tags.

Any insights ?

Thx

**Comment by TomSweeneyRedHat** [comment]
Just in case others stumble on this, please see: https://github.com/containers/podman/discussions/19796 for updates and status on this issue.
